warehouse/README.md

```diff
@@ -38,3 +39,20 @@ docker-compose up -d
 
 Then use any db client to connect to `localhost:5431` using the db credentials
 in `.env`.
+
+## Data Organization
+
+You can persist extract and transform outputs of any Kids First studies in the warehouse, using the ingest [library](https://github.com/kids-first/kf-lib-data-ingest) and [packages](https://github.com/kids-first/kf-ingest-packages).
+Upon ingest of a study, a new database named after the study's KF ID will be created with two schemas: 1) `Extract` and 2) `Transform`. The below figure illustrates the organization of ingested data:
+
+[Draw.io Source](https://drive.google.com/file/d/1IJGg6Dw9TeNzLzI-9fnUvo71Y6HL8UHN/view)
+
+![Data Organization](./static/img/data-organization.png)
```

Comment:
Change `AWS RDS` to `Kids First AWS` to make it more generic. Right now the warehouse is deployed on a EC2, later on we'll deploy it on RDS

Reply:
Modified!